src/option.pxi

```diff
@@ -352,6 +352,8 @@ cdef class Options:
                 out[field.name] = (self_val, other_val)
         return out
 
+    logical_types: tuple[LogicalType] = LOGICAL_TYPES + (GzipStringType,)
```

Comment:
Just add GzipStringType to the initial LOGICAL_TYPES definition, no need to do this here